hypershift-operator/controllers/machineconfigserver/assets/assets.go

```diff
@@ -0,0 +1,26 @@
+package assets
+
+import "embed"
+
+//go:embed machine-config-server/*
```

Comment:
We already purged templates from this operator, can we stick with the new pattern of defining all our resources in Go like the hosted control plane operator?